### DISCUSSION TITLE
Revisit drop-all-capabilities case insensitivity

### DISCUSSION THREAD

**Discussion by joebowbeer** [aquasecurity/trivy#6474] [Bugs]
### Description

From https://github.com/aquasecurity/trivy/discussions/4366#discussioncomment-9064887

### Desired Behavior

The trivy PSS check for strict drop all capabilities should only accept resources that comply with the PSS spec and PSA implementation.

In addition to not matching the spec, which is a problem in and of itself, the most likely problem that might arise in practice is that trivy will NOT catch this in CI/CD but a compliant admission controller WILL reject it when it is applied to the cluster.

See https://github.com/kubernetes/pod-security-admission/issues/11#issuecomment-1619967537

### Actual Behavior

Trivy accepts "all" in any mixture of upper and lowercase

### Reproduction Steps

Read https://github.com/aquasecurity/defsec/pull/1340

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A
```

### Operating System

N/A

### Version

```bash
N/A
```

### Checklist

- [X] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by simar7** [comment]
Thanks for starting the discussion. I'm not very familiar with the PSS spec but I believe @chen-keinan can take a look once he's back from his vacation as he had approved the previous PR.

**Comment by chen-keinan** [comment]
@joebowbeer thanks for the feedback , would you like to raise a PR ?
